src/Switch/index.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { CustomCheckboxContainer, CustomCheckboxInput } from '@reach/checkbox';
+import { MinervaProps, Box, systemProps } from '../layout';
+import VisuallyHidden from '../VisuallyHidden';
+// import { useTheme } from '../theme';
+
+const SwitchContainer = styled(Box)(
+  {
+    position: 'relative',
+    width: '75px',
+    display: 'inline-block',
+    verticalAlign: 'middle',
+    textAlign: 'left',
+  },
+  systemProps
+);
+
+export type ControlBoxProps = {
+  checked?: boolean;
+  theme?: any;
+};
+
+const ControlBox = styled(Box)<ControlBoxProps>(
+  (props: ControlBoxProps) => ({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderStyle: 'solid',
+    borderWidth: '1px',
+    borderRadius: '4px',
+    width: '16px',
+    height: '16px',
+    bg: 'primary',
+    marginRight: '8px',
+    transition: 'background-color 120ms ease 0s, box-shadow 250ms ease 0s',
+    borderColor: props.checked ? '#fff' : '#ecebed',
+    backgroundImage: `url("data:image/svg+xml;charset=utf-8,%3Csvg viewBox='0 0 16 16' fill='%23fff' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5.707 7.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4a1 1 0 00-1.414-1.414L7 8.586 5.707 7.293z'/%3E%3C/svg%3E")`,
```

Comment:
I'm not quite sure what this background image is doing in this context, I don't think I see any SVGs when looking at the switch